core/cat/looking_glass/cheshire_cat.py

```diff
@@ -501,30 +501,30 @@ def __call__(self, user_message_json):
 
     # TODO: remove this method in a few versions, current version 1.2.0
     def get_base_url():
-        """Allows the Cat expose the base url."""
+        """Allows the Cat exposing the base url."""
         log.warning("This method will be removed, import cat.utils tu use it instead.")
         return utils.get_base_url()
 
     # TODO: remove this method in a few versions, current version 1.2.0
     def get_base_path():
-        """Allows the Cat expose the base path."""
+        """Allows the Cat exposing the base path."""
         log.warning("This method will be removed, import cat.utils tu use it instead.")
         return utils.get_base_path()
 
     # TODO: remove this method in a few versions, current version 1.2.0
-    def get_plugin_path():
-        """Allows the Cat expose the plugins path."""
+    def get_plugins_path():
```

Comment:
Maybe leave it get_plugin_path for backwards compatibility?
 In any case we will remove these methods in a few releases.

Reply:
Yes, don't know how largely this method is used. I made it plural to avoid confusion with the method that provides only the current plugin path.
If you think this may break many things, I'll revert.